azext_edge/edge/providers/rpsaas/adr/params.py

```diff
@@ -1253,10 +1253,262 @@ def load_adr_arguments(self, _):
+            context.argument(
+                "asset_name",
+                options_list=["--asset", "-a"],
+                help="Asset name.",
+            )
+            context.argument(
+                "event_name",
+                options_list=["--name"],
+                help="Event name.",
+            )
+            context.argument(
+                "event_notifier",
+                options_list=["--event-notifier", "--en"],
+                help="Event notifier.",
+            )
+            context.argument(
+                "replace",
+                options_list=["--replace"],
+                help="Replace the event if another event with the same name is already present.",
+                arg_type=get_three_state_flag(),
+            )
+
+        with self.argument_context(f"iot ops ns asset {asset_type} event point") as context:
+            context.argument(
+                "event_name",
+                options_list=["--event", "-e"],
+                help="Event name.",
+            )
+            context.argument(
+                "datapoint_name",
+                options_list=["--name"],
+                help="Data point name.",
+            )
+            context.argument(
+                "data_source",
+                options_list=["--data-source", "--ds"],
+                help="Data source for the event data point.",
+            )
+            context.argument(
+                "replace",
+                options_list=["--replace"],
+                help="Replace the data point if another point with the same name is already present.",
+                arg_type=get_three_state_flag(),
+            )
+
+    with self.argument_context("iot ops ns asset custom dataset") as context:
+        context.argument(
+            "dataset_custom_configuration",
+            options_list=["--config"],
+            help="Custom dataset configuration as a JSON string or file path.",
+        )
+        context.argument(
+            "dataset_destinations",
+            options_list=["--destination", "--dest"],
+            help="Key=value pairs representing the destination for dataset. "
+            "Allowed arguments include: `key` for BrokerStateStore; `path` for Storage; or "
+            "`topic`, `retain`, `qos`, and `ttl` for MQTT.",
+            action="append",
+            nargs="+",
+        )
+
+    with self.argument_context("iot ops ns asset custom dataset point") as context:
+        context.argument(
+            "custom_configuration",
+            options_list=["--config"],
+            help="Custom data point configuration as a JSON string or file path.",
+        )
+
+    with self.argument_context("iot ops ns asset opcua dataset") as context:
+        context.argument(
+            "opcua_dataset_publishing_interval",
+            options_list=["--publish-int", "--pi"],
+            help="Publishing interval for datasets in milliseconds. Minimum: -1.",
```

Comment:
what is -1? not 0?

Reply:
this is from the service ppl - I believe -1 is realtime while 0 is constant or something like that